src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const successHandler = () => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+};
+
+const errorHandler = () => {
+  const message = document.createElement('div');
+
+  message.classList.add('message', 'error-message');
+  message.textContent = 'Promise was rejected!';
+  document.body.appendChild(message);
+};
```

Comment:
in the future you can create one function with params to control error/success notification